<launch> 
	<node pkg="image_transport_tutorial" type="my_publisher" name="bottom_cam_pub" required="true" args="1" />
	<node pkg="image_transport_tutorial" type="my_subscriber" name="bottom_cam_sub" respawn="true" args="1"/>
	<node pkg="image_transport_tutorial" type="python_sub.py" name="python_handler" respawn="true" output="screen"/>

	<!--
	<node pkg="image_proc" type="image_proc" name="top_rectifier" respawn="true" ns="camera1" output="screen"/>
	<node pkg="image_proc" type="image_proc" name="bottom_rectifier" respawn="true" ns="camera2" output="screen"/>
	-->
	
</launch>